### DISCUSSION TITLE
Change content tooltip của Expand All and Expand and Show/Hide Column

### DISCUSSION THREAD

**Discussion by duyhd** [KevinVandy/material-react-table#810] [General]
I use material-react-table and have the following problem. I want to change the text of the tooltips of the following images. Can anyone help me?
[
![Screenshot 2023-11-13 153424](https://github.com/KevinVandy/material-react-table/assets/49530713/6e620595-5f30-4540-89a1-caf0de760a1e)
![Screenshot 2023-11-13 153414](https://github.com/KevinVandy/material-react-table/assets/49530713/d1a8bbc3-cb3f-4ccf-84bd-b93afcbbab8c)
![Screenshot 2023-11-13 153245](https://github.com/KevinVandy/material-react-table/assets/49530713/9e9d341f-c5a3-469f-9633-29e8fa93fc9e)
](url)

**Comment by KevinVandy** [comment]
Read the docs https://www.material-react-table.com/docs/guides/localization

**Reply by duyhd** [reply]
Thanks. I fixed it
